_includes/head.html

```diff
@@ -18,7 +18,7 @@
   <meta property="og:title" content="{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}"/>
   <meta property="og:type" content="article"/>
   <meta property="og:url" content="{{ site.production_url }}{{ page.url | replace:'index.html',''}}"/>
-  <meta property="og:description" content="{% if page.description %}{{ page.description }} | {{ page.subtitle }}{% else %}{{ site.description }}{% endif %}" />
```

Comment:
We shouldn't remove this from the header, the problem should be fixed in the YAML front matter